frontend/src/component/feature/FeatureStrategy/FeatureStrategyConstraints/EditableConstraint.tsx

```diff
@@ -205,17 +206,19 @@ export const EditableConstraint: FC<Props> = ({
     onUndo,
     toggleInvertedOperator,
     toggleCaseSensitivity,
-    input,
     contextDefinition,
     constraintValues,
     constraintValue,
     setValue,
     setValues,
     setValuesWithRecord,
-    setError,
     removeValue,
-    error,
 }) => {
+    const { input } = useConstraintInput({
+        contextDefinition,
+        localConstraint,
+    });
```

Comment:
moving this here from the wrapper for now. I don't actually think we need this hook at all. I wanna spend the next PR looking at if we can get rid of that (and .. a lot of other stuff in those components).